src/NUnitFramework/framework/Attributes/CultureAttribute.cs

```diff
@@ -29,8 +29,7 @@
 namespace NUnit.Framework
 {
     /// <summary>
-    /// CultureAttribute is used to mark a test fixture or an
-    /// individual method as applying to a particular Culture only.
+    /// Marks a test fixture or method as applying to a specific Culture.
```

Comment:
Should this also mention assembly, since the attribute has `AttributeTargets.Assembly`?